docs/getting-started-inferencing.md

```diff
@@ -13,86 +10,79 @@ You are assumed to have deployed the llm-d inference stack from a guide, using t
-
-    ```bash
-    kubectl get gateway -n ${NAMESPACE}
-    NAME                                           CLASS      ADDRESS                                                                   PROGRAMMED   AGE
-    infra-inference-scheduling-inference-gateway   kgateway   af805bef3ec444a558da28061b487dd5-2012676366.us-east-1.elb.amazonaws.com   True         11m
-    infra-sim-inference-gateway                    kgateway   a67ad245358e34bba9cb274bc220169e-1351042165.us-east-1.elb.amazonaws.com   True         45s
-
-    GATEWAY_NAME=infra-inference-scheduling-inference-gateway
-    export ENDPOINT=$(kubectl get gateway ${GATEWAY_NAME} --no-headers -n ${NAMESPACE} -o jsonpath='{.status.addresses[0].value}')
-    ```
-  </TabItem>
-  <TabItem value="ingress" label="Ingress Controller">
-    > [!REQUIREMENTS]
-    > This requires that the release of the `llm-d-infra` chart must have `.ingress.enabled` set to `true`, and the `.gateway.service.type` to `ClusterIP`.
-    > This requires some load-balancer configuration for your cluster / ingress-controller. This could be either cloud-provider integration or something like MetalLB.
-
-    This is the most environment dependent of all the options, and can be tricky to set up. For more information on this see [our gateway customization docs](../docs/customizing-your-gateway.md#using-an-ingress). You should be able to get your endpoint from your ingress with the following:
-
-    ```bash
-    export ENDPOINT=$(kubectl get ingress --no-headers -o jsonpath='{.items[].status.loadBalancer.ingress[0].ip}')
-    ```
-
-    **_NOTE:_** This command assumes you have one ingress in your given `${NAMESPACE}`, if you have multiple, it will just grab one. Therefore, in the case you do have multiple gateways, you should find the correct gateway and target that specifically:
-
-    ```bash
-    kubectl get ingress -n ${NAMESPACE}
-    NAME                                           CLASS     HOSTS   ADDRESS         PORTS   AGE
-    infra-inference-scheduling-inference-gateway   traefik   *       166.19.16.120   80      21m
-    infra-sim-inference-gateway                    traefik   *       166.19.16.132   80      7m
-
-    INGRESS_NAME=infra-inference-scheduling-inference-gateway
-    export ENDPOINT=$(kubectl get ingress ${GATEWAY_NAME} --no-headers -n ${NAMESPACE} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-    ```
-  </TabItem>
-</Tabs>
+**_NOTE:_** If you're unsure which to use—start with port-forward as it's the most reliable and easiest. For anything shareable, use Ingress/Route. Use LoadBalancer if your provider supports it and you just need raw L4 access.
+
+<!-- TABS:START -->
+
+<!-- TAB:Port-forward (Cluster Internal):default -->
+For gateway providers that install into the cluster you can port forward to the gateway deployment directly.
```

Comment:
Let's add title `##Port-forward (Cluster Internal)` here otherwise it's really hard to navigate on github

Reply:
All set

<img width="718" height="316" alt="Screenshot 2025-11-21 at 2 18 00 PM" src="https://github.com/user-attachments/assets/7d4c47d5-8bc9-4819-8cec-3087e78e6222" />